components/landing/FeaturedTopic.tsx

```diff
@@ -24,7 +24,7 @@ export default function FeaturedTopic({ title, prayerPoints, featuredImg }: Prop
 
   return (
     <Container className="d-flex justify-content-center">
-      <Card className="my-4 shadow d-lg-flex flex-lg-row">
+      <Card className="my-4 shadow d-md-flex flex-md-row">
```

Comment:
For example, if you look at this component on the homepage and shrink it a bit, you'll see that this component isn't quite right at the md-lg range. I think this should be kept at d-lg-flex and flex-lg-row

Reply:
yeah so I think we need to tweak other things, but the point isn't to cater for the md-lg range, it's to cater for the range from tablet to desktop (md -> xl)